headers/types/dungeon_mode/dungeon_mode.h

```diff
@@ -812,12 +984,17 @@ struct dungeon_generation_info {
     undefined field_0x19;
     uint16_t floor_generation_attempts; // 0x1A: Number of attempts at floor layout generation
     struct tile tiles[32][56];          // 0x1C
-    struct position player_spawn_pos;   // 0x8C1C: Position of the player spawn
+    struct position team_spawn_pos;     // 0x8C1C: Position of the team spawn
     struct position stairs_pos;         // 0x8C20: Position of the stairs spawn
     // 0x8C24: Position of the Hidden Stairs spawn, or (-1, -1) if no Hidden Stairs
     struct position hidden_stairs_pos;
+    struct position leader_spawn_pos; // 0x8C28: Spawn position of the leader
+    struct position ally_1_spawn_pos; // 0x8C2C: Spawn position of the second team member
+    struct position ally_2_spawn_pos; // 0x8C30: Spawn position of the third team member
+    struct position ally_3_spawn_pos; // 0x8C34: Spawn position of the fourth team member
+    // There's another 6 potential positions right after these
```

Comment:
I'll reword that to make it clearer. I put that comment because when I looked at this area in the RAM, it seemed like there were 6 more unitinialized positions, so there might be 10 positions in total. But right now I can't confirm if those other 6 words are part of the struct or not, so that's why the comment is there.

Reply:
Okay, we can always expand the struct later if needed.